secondLayout.js

```diff
@@ -0,0 +1,317 @@
+// Getting the canvas from html page
+let canvas = document.getElementById("canvas");
+let ctx = document.getElementById("canvas").getContext("2d");
+let bodyCtx = document.querySelector("body");
+
+window.onload = () => {
+  firstPage();
+};
+
+function startGame() {
+  isOutOfFirstPage = true;
+  drawGameBoard();
+}
+
+// to draw the first page with bg and text on canvas
+function firstPage() {
+  drawFirstBg();
+  let newText = new text(gameStory, 100, 150, 35);
+  newText.update(100);
+}
+
+// function to remove objects from the objects array and from the canvas when they reach the end of the canvas
+function removeObject(objectsArr) {
```

Comment:
I guess since `objectsArr` is global anyways you don't have to pass it into the function creating a parameter that is called the same as the global var, which will lead to confusion